File: service/src/main/java/coding/challenge/core/model/Phrase.java
package coding.challenge.core.model;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.Id;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * Phrase entity.
 */
@Entity
@Data
@NoArgsConstructor
public class Phrase {

    /**
     * Phrase identifier.
     */
    @Id
    @GeneratedValue
    private long id;

}

File: service-kt/src/main/kotlin/coding/challenge/core/model/Phrase.kt
package coding.challenge.core.model

import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.Id

/**
 * Phrase entity.
 */
@Entity
data class Phrase(
    /**
     * Phrase identifier.
     */
    @Id
    @GeneratedValue
    val id: Long? = null,
)
